### PR TITLE
Userstarsrepo

### PR DESCRIPTION
**Pull request by Erriep**
feature API five bigger stars repository filter and style + add axios…

### CODE REVIEW

**Review by Hacene010 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "react-dom": "^17.0.2",
     "react-scripts": "4.0.3",
     "sass": "^1.32.8",
+    "styled-components": "^5.2.3",
```
Comment (LoicBrassart) [inline package.json]: One or the other, but not both ;-)

File: src/components/UserStarsRepo.scss [hunk 2]
```diff
@@ -0,0 +1,120 @@
+.UserStarsRepo {
```
Comment (LoicBrassart) [inline src/components/UserStarsRepo.scss:1]: Inconcistent with the addition of `styled-components`

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Which styling tool do you want to use ?

**Review by Hacene010 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**